src/client/components/Header/Header.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Header.scss';
+
+const Header = () => (
+  <React.Fragment>
```

Comment:
Fragment is used when you have more than one direct children, it's not needed here at the moment